lms/templates/dashboard.html

```diff
@@ -84,16 +86,15 @@ <h2 class="header-courses">${_("Current Courses")}</h2>
         <% is_course_blocked = (enrollment.course_id in block_courses) %>
         <% course_verification_status = verification_status_by_course.get(enrollment.course_id, {}) %>
         <% course_requirements = courses_requirements_not_met.get(enrollment.course_id) %>
-        <%include file='dashboard/_dashboard_course_listing.html' args="course_overview=enrollment.course_overview, enrollment=enrollment, show_courseware_link=show_courseware_link, cert_status=cert_status, credit_status=credit_status, show_email_settings=show_email_settings, course_mode_info=course_mode_info, show_refund_option=show_refund_option, is_paid_course=is_paid_course, is_course_blocked=is_course_blocked, verification_status=course_verification_status, course_requirements=course_requirements, dashboard_index=dashboard_index, share_settings=share_settings, user=user" />
-      % endfor
-
-      % if settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
-        % for ccx, membership, course in ccx_membership_triplets:
+        % if isinstance(enrollment.course_id, CCXLocator):
+          <% ccx = get_ccx_from_ccx_locator(enrollment.course_id) %>
           <% show_courseware_link = ccx.has_started() %>
           <% is_course_blocked = False %>
           <%include file='ccx/_dashboard_ccx_listing.html' args="ccx=ccx, membership=membership, course_overview=enrollment.course_overview, show_courseware_link=show_courseware_link, is_course_blocked=is_course_blocked" />
```

Comment:
Is it possible to collapse this further so that CCX courses are rendered on the dashboard by the normal `_dashboard_course_listing.html` template? What are the important deltas to preserve in terms of presentation between CCX courses and normal ones?